src/ui/shared/requests/useNativeBalance.ts

```diff
@@ -29,10 +31,18 @@ function useNativeAddressPosition({
   );
 
   return useMemo(() => {
+    const nativePositions = value?.positions?.filter(
+      (item) =>
+        item.chain === chain.toString() &&
+        item.type === 'asset' &&
+        !item.protocol
```

Comment:
As I understand, if item.type === asset, it can't be inside protocol. So this filter can be simplified to 
` (item) => item.chain === chain.toString() && item.type === 'asset'`

Reply:
Oh I see. Do we really get position with type: asset and non-empty protocol?

Reply:
Yeah, this was exactly the reason why I had to add this condition

Reply:
Wow. Just faced this. Approve then!